charts/loki/values.yaml

```diff
@@ -318,3 +318,7 @@ alerting_groups: []
 #    - alert: HighThroughputLogStreams
 #      expr: sum by(container) (rate({job=~"loki-dev/.*"}[1m])) > 1000
 #      for: 2m
+
+useExistingAlertingGroup:
+  enabled: false
+  configmapName: ""
```

Comment:
Check failure on line 324 in charts/loki/values.yaml

GitHub Actions
/ lint-test

charts/loki/values.yaml#L324

[new-line-at-end-of-file] no new line character at the end of file

Reply:
done